pulse_actions/handlers/treeherder_add_new_jobs.py

```diff
@@ -62,12 +66,26 @@ def on_event(data, message, dry_run, treeherder_host, acknowledge, **kwargs):
                                                                           treeherder_link))
 
     LOG.info("New jobs requested by %s for %s" % (requester, treeherder_link))
-    LOG.info("List of builders:")
-    for b in buildernames:
+    LOG.info("List of requested jobs:")
+    for b in requested_jobs:
         LOG.info("- %s" % b)
 
+    # Handle TC tasks separately
+    task_labels = [x for x in requested_jobs if x.startswith('TaskLabel==')]
+    buildernames = list(set(requested_jobs) - set(task_labels))
+
     buildernames = filter_invalid_builders(buildernames)
 
+    # Scheduling TaskCluster jobs
+    # Make sure that decision task id is not null and task_labels are there to schedule
+    if task_labels and len(decision_task_id):
+        try:
+            schedule_action_task(decision_task_id=decision_task_id,
```

Comment:
Let's add in here something like this:
```python
mgr = TaskClusterManager(dry_run=dry_run)
mgr.schedule_action_task(...)
```
This will require adjusting mozci.
Move this function (and make appropiate adjustments):
https://github.com/mozilla/mozilla_ci_tools/blob/master/mozci/taskcluster/tc.py#L438
inside of the class.